docs/src/dataDisplay/list/list.jsx

```diff
@@ -95,24 +94,25 @@ export default class ElementsList extends React.Component {
             return (
                 <div>
                     <Card>
-                        <Header size="large">Props</Header>
+                        <Typography size="large">Props</Typography>
 
                         <TableProps props={props} />
                     </Card>
 
                     {/* List */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading variant="h2">
                         List
-                        <Header.Subheader>
-                            A basic card has a faint border wrapping it's contents. It's box model is fluid so that it can adapt to it's parent container's width.
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
+
+                    <Typography variant="body1">
+                        A basic card has a faint border wrapping it's contents. It's box model is fluid so that it can adapt to it's parent container's width.
```

Comment:
All of the instances of "it's" in here should be "its" (no apostrophe).
Why is this talking about a Card on the List documentation page?  Is that on purpose?  Or was this a copy-and-paste error?

Reply:
Looks like a copy and paste error.

Reply:
Seems to have been removed.  Resolving.